src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.html

```diff
@@ -7,10 +7,8 @@
         Select Mission & Runtime
       </h1>
       <p>
-        Start by selecting a runtime or a mission. Valid pairings are narrowed based on your initial selection.
-        Missions are preconfigured, functioning applications that demonstrate a fundamental aspect of modern application
-        development running in an environment similar to production. These can be used as a proof of technology
-        demonstration, a teaching tool, or even a sandbox for understanding how to develop applications.
+        Choose a mission and runtime to create a booster or sample application.You set up a booster to generate 
```

Comment:
Per design, this should not stretch across the entire page - it should only be over the "Mission" card.